github/issues_timeline_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestIssuesService_ListIssueTimeline(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/issues/1/timeline", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypeTimelinePreview)
+		testFormValues(t, r, values{
+			"page":     "1",
+			"per_page": "2",
+		})
+		fmt.Fprint(w, `[{"id":1}]`)
+	})
+
+	opt := &ListOptions{Page: 1, PerPage: 2}
+	events, _, err := client.Issues.ListIssueTimeline("o", "r", 1, opt)
+
+	if err != nil {
```

Comment:
There's a blank line between assigning `err` and checking it. I suggest removing it. It's not needed, and makes the code less readable (the blank line suggests a two sections of code are separate concerns).

I can see there are a few other rare places where this is done in this code base, but the vast majority of code does not, so this would also increase consistency.